generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -466,44 +466,66 @@ Scenario: Running a 'matchingRegex' request alongside a granularTo constraint sh
       | "10" |
 
 Scenario: Running a 'matchingRegex' request alongside a after constraint should be successful
-     Given foo is matching regex /[0-z]{2}/
+     Given foo is matching regex /[a-z]{2}/
```

Comment:
Is it worth just having in `[a-e] {2}` here? As this will help to further cut down compute time

Reply:
changed